Orange/widgets/data/owcontinuize.py

```diff
@@ -378,6 +378,18 @@ def __call__(self, data):
         )
         return newdomain
 
+    def __repr__(self):
+        return "DomainContinuizer({}{}{}{})".format(
+            "zero_based=False, ".format(self.zero_based) if not
+                self.zero_based else "",
+            "multinomial_treatment=Continuize.{}, ".format(repr(self.multinomial_treatment)) if
+                self.multinomial_treatment != Continuize.Indicators else "",
+            "continuous_treatment=Continuize.{}, ".format(repr(self.continuous_treatment)) if
+                self.continuous_treatment != Continuize.Leave else "",
+            "class_treatment=Continuize.{}, ".format(repr(self.class_treatment)) if
+                self.class_treatment != Continuize.Leave else ""
```

Comment:
I think this might work with `Reprable` also. The only botherseome thing, on multiple counts, is the use of `__new__`. I'm totally _for_ stripping `data` arg from `__new__` (now `__init__`) and then replacing all occurrences of `DomainContinuizer(data, ...)` with `DomainContinuizer(...)(args)`, i.e. construct the continuizer object first, then call it to transform the data, like the rest of preprocessors do.